tests/main/security-dev-input-event/task.yaml

```diff
@@ -0,0 +1,68 @@
+summary: Ensure that /dev/input/event* is denied by default.
+
+details: |
+    The default policy disallows access to /dev/input/event*.
+
+    The joystick interface disallows access to /dev/input/event* for
+    non-joysticks.
+
+    The test checks the snap is not able to access /dev/input/event* with or
+    without the joystick interface connected. We do this since the
+    /dev/input/event* devices are sensitive and because the joystick interface
+    adds a /dev/input/event* AppArmor glob rule that relies entirely on the
+    device cgroup for enforcement.
+
+prepare: |
+    echo "Given the test-snapd-event snap is installed"
+    snap install test-snapd-event
+
+restore: |
+    rm -f call.error
+    snap remove test-snapd-event
+
+execute: |
+    echo "The joystick plug is not connected by default"
+    snap interfaces -i joystick | MATCH "\- +test-snapd-event:joystick"
+
+    if [ "$(snap debug confinement)" != "strict" ]; then
```

Comment:
That would be nice if we can exit after testing the behavior when the snap is connected to the interface. Most of the tests are following the next pattern:
1. check default connection
2. if not connected by default then connect
3. test intertace works when it is connected
4. disconnect
5. exit in case confinement is not strict
6. check the errors when the interface is not connected